mindjunkies/lecture/models.py

```
@@ -60,11 +60,12 @@ class LectureVideo(BaseModel):
     )
-    lecture = models.ForeignKey('Lecture', on_delete=models.CASCADE, related_name='videos')
-    video_file = models.FileField(upload_to='lecture_videos/')
-    video_title = models.TextField(null=True, blank=True)
-    thumbnail = models.ImageField(upload_to="thumbnails", null=True, blank=True)
-    hls = models.CharField(max_length=500, blank=True, null=True)
+
+    lecture=models.ForeignKey('Lecture',on_delete=models.CASCADE,related_name='videos')
+    video_file=models.FileField(upload_to='lecture_videos/')
+    video_title=models.CharField(max_length=255)
+    thumbnail = models.ImageField(upload_to="thumbnails",null=True,blank=True)
+    hls = models.CharField(max_length=500,blank=True,null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)